chia/data_layer/data_layer_wallet.py

```diff
@@ -1272,10 +1272,9 @@ async def finish_graftroot_solutions(offer: Offer, solver: Solver) -> Offer:
                         asserted_root: Optional[str] = None
                         proofs_of_inclusion = []
                         for value in values:
-                            for root in solver["proofs_of_inclusion"].info:
-                                proof: Tuple[int, List[bytes32]] = tuple(  # type: ignore
-                                    solver["proofs_of_inclusion"][root]
-                                )
+                            for proof_of_inclusion in solver["proofs_of_inclusion"]:
+                                root = proof_of_inclusion[0]
+                                proof = tuple(proof_of_inclusion[1:])
```

Comment:
This is still just playing on the `Any` nature of the underlying data, but might satisfy things.

```suggestion
                                proof: Tuple[int, List[bytes32]] = tuple(proof_of_inclusion[1:])
```

Reply:
This is wrong, I should have removed it before heading out...  sorry.